src/lv_widgets/lv_chart.c

```diff
@@ -702,6 +732,64 @@ lv_chart_axis_t lv_chart_get_series_axis(lv_obj_t * chart, lv_chart_series_t * s
 
     return ser->y_axis;
 }
+
+lv_point_t lv_chart_get_cursor_point(lv_obj_t * chart, lv_chart_cursor_t * cursor)
+{
+    LV_ASSERT_NULL(cursor);
+    LV_UNUSED(chart);
+
+    return cursor->point;
+}
+
+uint16_t lv_chart_get_nearest_index_from_coord(lv_obj_t * chart, lv_point_t coord)
+{
+    lv_area_t series_area;
+    get_series_area(chart, &series_area);
+
+    lv_coord_t w = lv_area_get_width(&series_area);
+    coord.x -= lv_obj_get_style_pad_left(chart, LV_CHART_PART_BG);
+
+    uint16_t id = 0;
+    lv_chart_ext_t * ext = lv_obj_get_ext_attr(chart);
+
+    if(coord.x < 0) {
+	    id = 0;
+    } else if(coord.x > w) {
+	    id = ext->point_cnt - 1;
+    } else {
+	    id = coord.x * (ext->point_cnt - 1) / w;
```

Comment:
Just asking: is it working with line and column type too?

Reply:
I forgot the column type, but it seems it works well for line type.